server/website/website/views.py

```diff
@@ -816,3 +816,26 @@ def get_timeline_data(request):
             data_package['timelines'].append(data)
 
     return HttpResponse(JSONUtil.dumps(data_package), content_type='application/json')
+
+
+# get the lastest result
+def give_result(request, upload_code):  # pylint: disable=unused-argument
+    try:
+        session = Session.objects.get(upload_code=upload_code)
+    except Session.DoesNotExist:
+        LOG.warning("Invalid upload code: %s", upload_code)
+        return HttpResponse("Invalid upload code: " + upload_code)
+    results = Result.objects.filter(session=session)
+    lastest_result = results[len(results) - 1]
```

Comment:
Want a more pythonic way of getting the last element in a array? Use this:
```
results[-1]
```

I'd also defensively check if `results` is not empty:

```
        if results:
           latest_result = ...
```

Reply:
[-1] returns an error, so I use [len(results)-1]